### ISSUE TITLE
Sporadic xcffib.xproto.AtomError

### ISSUE THREAD

**Issue by jspam** [talonvoice/talon#387]
On Talon 0.2.1 on Linux, I am getting the following message in the log every time I switch windows:

```
xcffib.xproto.AtomError
2021-09-29 07:26:07 ERROR failed to get window info (update_windows)
   12:                     lib/python3.9/threading.py:912| 
   11:                     lib/python3.9/threading.py:954| 
   10:                     lib/python3.9/threading.py:892| 
    9:                         talon/linux/xclient.py:429| 
    8:                         talon/linux/xclient.py:325| # [stack splice]
    7:                         talon/linux/xclient.py:201| 
    6:                         talon/linux/xclient.py:188| 
    5:                         talon/linux/xclient.py:123| 
    4: lib/python3.9/site-packages/xcffib/__init__.py:332| 
    3: lib/python3.9/site-packages/xcffib/__init__.py:571| 
    2: lib/python3.9/site-packages/xcffib/__init__.py:658| 
    1: lib/python3.9/site-packages/xcffib/__init__.py:649|
```
Rules which depend on a window's class are not activated.

Several restarts of Talon make the problem go away eventually (until the next reboot).

My window manager is i3, I am on Ubuntu 20.04.3 LTS. I did not experience the issue before the 0.2.1 upgrade (which I'd like to keep because of the new speech model's amazing quality …).

**Comment by lunixbochs** [comment]
Hey, I reached out on Slack about this, I have a fix for it I'd like you to test so I can confidently ship it in the public release.

**Comment by lunixbochs** [comment]
fixed in v0.2.3
